### ISSUE TITLE
add some extra letters to the moh mapping

### ISSUE THREAD

**Issue by joanise** [roedoejet/g2p#227]
Reported by Akwiratékha':
So as I mentioned briefly in the workshop that Kanien'kéha does have additional sounds that are due to borrowings from French and English.
P, B, and M, as well as the vowel U [u].
Aprám = Abraham. Borakén: = Cord du roi. Makhró: = an dummy haha. We have a lot.

**Comment by joanise** [comment]
p, b, m and u are valid IPA, so if they occur they'll just stay unchanged. I don't think we need to patch anything. 

Confirming with Akwiratékha' whether this is already correct, but I believe it is:
```
$ g2p convert --tok "Borakén Makhró Aprám aonsákken' mu" moh moh-ipa
boraɡʌ̃́ makhró aprám aũsákɡʌ̃ʔ mu
```

**Comment by joanise** [comment]
517039f13a adds explicit mappings. They're no-ops, but the can serve as documentation. Not sure yet if I want to PR this or  not, though.

**Comment by joanise** [comment]
Fixed in #234
